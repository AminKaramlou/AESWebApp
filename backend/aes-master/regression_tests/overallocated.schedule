1: A
2: A
