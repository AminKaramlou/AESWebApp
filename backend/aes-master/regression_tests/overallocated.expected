Schedule is not feasible because:
 -  Job A is allocated to multiple machines {1, 2}
Schedule does satisfies user fixed decisions
 -  All jobs satisfy all fixed decisions
Schedule is efficient
 -  All jobs satisfy single and pairwise exchange properties
